### PR TITLE
feat(crons): Add sidebar with cron monitor information in details page

### LINKED ISSUE #47919: [FE] Display monitor details on right hand side of details

### PR DESCRIPTION
**Pull request by davidenwang**
Before:
image

After:
image

Fixes https://github.com/getsentry/sentry/issues/47919

### CODE REVIEW

File: static/app/views/monitors/components/detailsSidebar.tsx [hunk 1]
```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {SectionHeading} from 'sentry/components/charts/styles';
```
Comment (evanpurkhiser) [inline static/app/views/monitors/components/detailsSidebar.tsx:4]: Why is this in charts 😬

File: static/app/views/monitors/components/detailsSidebar.tsx [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+import Clipboard from 'sentry/components/clipboard';
+import {KeyValueTable, KeyValueTableRow} from 'sentry/components/keyValueTable';
+import Text from 'sentry/components/text';
+import TimeSince from 'sentry/components/timeSince';
+import {IconCopy} from 'sentry/icons';
+import {t, tn} from 'sentry/locale';
+import space from 'sentry/styles/space';
```
Comment (evanpurkhiser) [inline static/app/views/monitors/components/detailsSidebar.tsx:11]: I think you need to import as `{space}`

File: static/app/views/monitors/components/detailsSidebar.tsx [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+  align-items: center;
+  gap: ${space(1)};
+`;
+
+const MonitorSlug = styled('div')`
+  display: grid;
+  grid-template-columns: 1fr max-content;
+  align-items: center;
+  gap: ${space(0.5)};
+  cursor: pointer;
```
Comment (evanpurkhiser) [inline static/app/views/monitors/components/detailsSidebar.tsx:139]: Why pointer?

**Review by evanpurkhiser — APPROVED** [review]

File: static/app/views/monitors/components/detailsSidebar.tsx (hunk 1 above)
Comment (davidenwang) [reply static/app/views/monitors/components/detailsSidebar.tsx:4]: Lol yea, seems like most of the times this is used in charts but definitely not only charts, should probably just bring it out into components.

File: static/app/views/monitors/components/detailsSidebar.tsx (hunk 2 above)
Comment (davidenwang) [reply static/app/views/monitors/components/detailsSidebar.tsx:11]: ahh gotcha yea will do

File: static/app/views/monitors/components/detailsSidebar.tsx (hunk 3 above)
Comment (davidenwang) [reply static/app/views/monitors/components/detailsSidebar.tsx:139]: just to show that it's clickable and copyable

image

Comment (evanpurkhiser) [reply static/app/views/monitors/components/detailsSidebar.tsx:139]: Do we do the same thing in other places?

Comment (evanpurkhiser) [reply static/app/views/monitors/components/detailsSidebar.tsx:139]: Why doesn't the clipboard component do this for us actually?

Comment (davidenwang) [reply static/app/views/monitors/components/detailsSidebar.tsx:139]: seems like a bit of the same thing in other places https://github.com/getsentry/sentry/blob/e4ac987d6bc1f7f196d332699f56e5093ea5567b/static/app/components/version.tsx#L185

not sure why clipboard doesn't do this for us though

Comment (davidenwang) [reply static/app/views/monitors/components/detailsSidebar.tsx:139]: lol yea this is just everywhere, https://github.com/getsentry/sentry/blob/952e4de9a068694babaf795c2949fedbdd1ba606/static/app/components/events/userFeedback.tsx#L70

**Pull request merged**